app/bundles/EmailBundle/Controller/PublicController.php

```diff
@@ -220,17 +220,16 @@ public function unsubscribeAction($idHash)
                     // check if tokens are present
                     $savePrefsPresent = false !== strpos($html, 'data-slot="saveprefsbutton"') ||
                                         false !== strpos($html, BuilderSubscriber::saveprefsRegex);
-                    $frequencyPresent = false !== strpos($html, 'data-slot="channelfrequency"') ||
-                                        false !== strpos($html, BuilderSubscriber::channelfrequency);
-                    $tokensPresent = $savePrefsPresent && $frequencyPresent;
-                    if ($tokensPresent) {
+                    if ($savePrefsPresent) {
                         // set custom tag to inject end form
                         // update show pref center slots by looking for their presence in the html
                         $params = array_merge(
                             $viewParameters,
                             [
                                 'form'                         => $formView,
                                 'custom_tag'                   => '<a name="end-'.$formView->vars['id'].'"></a>',
+                                'showContactFrequency'         => false !== strpos($html, 'data-slot="channelfrequency"') ||
```

Comment:
It's work good but would not it be better for consistency if the code of this line is formatted like the others, in one line?
```
'showContactFrequency' => false !== strpos($html, 'data-slot="channelfrequency"') || false !== strpos($html, BuilderSubscriber::channelfrequency)
```

Or for more readability like line 221?
```
'showContactFrequency' => false !== strpos($html, 'data-slot="channelfrequency"') ||
                          false !== strpos($html, BuilderSubscriber::channelfrequency)
```

Reply:
Depends what you want.
There is pre-hook during the commit with formatting that passes the code.
But I change it.